.jshintrc

```diff
@@ -23,7 +23,7 @@
   "undef"         : true,     // true: Require all non-global variables to be declared (prevents global leaks)
   "unused"        : false,    // true: Require all defined variables be used
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
-  "maxparams"     : 10,       // {int} Max number of formal params allowed per function
+  "maxparams"     : 12,       // {int} Max number of formal params allowed per function
```

Comment:
@ZitaNemeckova I'm not so sure about this change.

Are we going to bend this rule every time the ngInject params cap reaches the max value just to make the tests pass?
Soon enough, we would have a requirement for 13 and then we'll change it 13. 
In that case why even have this rule?

@chriskacerguis @AllenBW What do you think about this?